readest: 0.9.36 -> 0.9.38

ddogfoodd opened the pull request:
Changelog: https://github.com/readest/readest/releases/tag/v0.9.38
Diff: https://github.com/readest/readest/compare/v0.9.36...v0.9.38

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

ddogfoodd committed:
readest: 0.9.36 -> 0.9.37

Changelog: https://github.com/readest/readest/releases/tag/v0.9.37
Diff: https://github.com/readest/readest/compare/v0.9.36...v0.9.37

eljamm reviewed (approved):
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 401550`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>readest</li>
  </ul>
</details>

ddogfoodd committed:
readest: 0.9.37 -> 0.9.38

Changelog: https://github.com/readest/readest/releases/tag/v0.9.38
Diff: https://github.com/readest/readest/compare/v0.9.37...v0.9.38

eljamm commented:
> `NEXT_TELEMETRY_DISABLED`

Interesting. Guess we can [wrap the executable](https://nixos.org/manual/nixpkgs/unstable/#fun-makeWrapper) with the env var.

eljamm commented:
Yeah, I don't think that will work as I imagine the file won't be created on the users' machines. Let's just wrap the executable, then.

eljamm commented:
> > Yeah, I don't think that will work as I imagine the file won't be created on the users' machines. Let's just wrap the executable, then.
> 
> Hm but we would have to wrap `pnpm` not `readest`, no? That is if I understand correctly that the telemetry belongs to building not executing.

Ah, I misunderstood. I thought this was some runtime telemetry. Yeah, we don't have to do this then, since networking is disabled by default on Linux builds (because of the [sandbox](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#tested-using-sandboxing)).
